custom/icds_reports/utils/number_of_awws_who_were_active_from_1st_jan_2019_to_30th_Jun_2019.sql

```diff
@@ -0,0 +1,23 @@
+SELECT COUNT(DISTINCT awc_id) FROM "ucr_icds-cas_static-usage_forms_92fbe2aa" WHERE form_time BETWEEN '2019-01-01' AND '2019-06-30';
```

Comment:
the PR title says 30th Jan but query says 30th June, which is correct?

Reply:
Typo error fixed